docs/releases/3.0.txt

```diff
@@ -235,6 +235,12 @@ Dropped support for PostgreSQL 9.4
 Upstream support for PostgreSQL 9.4 ends in December 2019. Django 3.0 supports
 PostgreSQL 9.5 and higher.
 
+Dropped support for Oracle 12.1
+-------------------------------
+
+Upstream support for Oracle 12.1 ends in July 2021. Django 2.2 will be
+supported until April 2022. Django 3.0 officially supports Oracle 18c.
```

Comment:
As someone unfamiliar with Oracle version number, it's unclear to me what the status of Oracle 12.2 is?